### PR TITLE
yandex-browser: init at 21.5.3.753-1

### PR DESCRIPTION
**Pull request by dan4ik605743**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cole-h** [comment]
@ofborg eval

(Sorry, ofborg's certs recently had an issue, meaning that it was unable to
react to PRs. This has been fixed, but all PRs filed in this period will need
ofborg to be manually kicked off.)

**Comment by dan4ik605743** [comment]
@SuperSandro2000 Thank you so much. I have 3 more packages weigh in PR. could you see them too.

File: pkgs/applications/networking/browsers/yandex-browser/default.nix [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    nss
+    pango
+    stdenv.cc.cc.lib
+  ];
+
+  unpackPhase = ''
+    mkdir -p $TMP/ya $out/bin
+    cp $src $TMP/ya.deb
+    ar vx ya.deb
+    tar --no-overwrite-dir -xvf data.tar.xz -C $TMP/ya/
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/yandex-browser/default.nix:119]: We normally use ``dpkg -x $src $TMP`` for unpacking debs. Would that work here, too?

File: pkgs/applications/networking/browsers/yandex-browser/default.nix [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+let
+  mirror = "http://repo.yandex.ru/yandex-browser/deb/pool/main/y";
+in
+
+stdenv.mkDerivation rec {
+  pname = "yandex-browser";
+  version = "21.5.3.753-1";
+
+  src = fetchurl {
+    url = "${mirror}/${pname}/${pname}_${version}_amd64.deb";
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/yandex-browser/default.nix:60]:
```suggestion
stdenv.mkDerivation rec {
  pname = "yandex-browser";
  version = "21.5.3.753-1";

  src = fetchurl {
    url = "http://repo.yandex.ru/yandex-browser/deb/pool/main/y/${pname}/${pname}_${version}_amd64.deb";
```

File: pkgs/applications/networking/browsers/yandex-browser/default.nix [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    libpulseaudio.out
+    (lib.getLib systemd)
+  ];
+
+  meta = with lib; {
+    description = "Yandex Web Browser";
+    homepage = "https://browser.yandex.ru/";
+    license = licenses.unfree;
+    maintainers = with maintainers; [ dan4ik605743 ];
+    platforms = platforms.linux;
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/yandex-browser/default.nix:137]:
```suggestion
    platforms = [ "x86_64-linux" ];
```

File: pkgs/applications/networking/browsers/yandex-browser/default.nix (hunk 1 above)
Comment (dan4ik605743) [reply pkgs/applications/networking/browsers/yandex-browser/default.nix:119]: @SuperSandro2000 You thought I wasn't trying. There is an error with the files. The --no-overwrite-dir flag is required. Therefore, you need to leave it as it is. Other options 0.

**Comment by dan4ik605743** [comment]
@SuperSandro2000 Please check my PR

**Pull request merged**